Plugins/NativeServerPlugin/StreamingNativeServerPlugin.vcxproj

```diff
@@ -185,26 +185,33 @@
     <ClCompile Include="src\conductor.cpp" />
     <ClCompile Include="src\custom_video_capturer.cpp" />
     <ClCompile Include="src\defaults.cpp" />
-    <ClCompile Include="src\default_data_channel_observer.cpp" />
-    <ClCompile Include="src\video_helper.cpp" />
+    <ClCompile Include="src\input_data_channel_observer.cpp" />
+    <ClCompile Include="src\buffer_renderer.cpp" />
+    <ClCompile Include="src\render_service.cpp" />
+    <ClCompile Include="src\service_base.cpp" />
   </ItemGroup>
   <ItemGroup>
     <ClInclude Include="inc\conductor.h" />
     <ClInclude Include="inc\custom_video_capturer.h" />
     <ClInclude Include="inc\defaults.h" />
-    <ClInclude Include="inc\default_data_channel_observer.h" />
+    <ClInclude Include="inc\input_data_channel_observer.h" />
     <ClInclude Include="inc\plugindefs.h" />
     <ClInclude Include="inc\flagdefs.h" />
     <ClInclude Include="inc\macros.h" />
-    <ClInclude Include="inc\pch.h" />
-    <ClInclude Include="inc\video_helper.h" />
+    <ClInclude Include="inc\buffer_renderer.h" />
+    <ClInclude Include="inc\service\render_service.h" />
+    <ClInclude Include="inc\service\service_base.h" />
+    <ClInclude Include="inc\service\thread_pool.h" />
     <ClInclude Include="pch.h" />
   </ItemGroup>
   <ItemGroup>
     <None Include="exports.props" />
     <None Include="nvEncConfig.json">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
+    <None Include="serverConfig.json">

```

Comment:
would it make sense to include the config values from this file in our existing config file? i believe that each config file adds a fair bit of complexity for a developer or person trying to use our stuff, and i'd love to keep them as consolidated as possible

Reply:
This is what @tylergibson refactored in his service branch so I just keep it until now. However, you make a good point here. What do you think Tyler?

Reply:
@tylergibson